main.js

```diff
@@ -1,4 +1,28 @@
 // Remember your promises
 // Go back to the lesson if you feel lost
 // Read the docs well, they usually have pretty well documented examples
-// Consider making your fetch work with a button
+
+const charactersList = $('.item');
+function getcharacters  () {
+
+    for (let i = 0; i <= 4; i++) {
+        const URL = 'https://rickandmortyapi.com/api/character/';
+
+    axios({ url: URL +i+1, method: "get" })
```

Comment:
`URL +i+1` output will be 01, 11, 21, 31, 41
to get the desired output should be
`URL +(i+1)`